src/app/zap-templates/zcl/data-model/chip/washer-controls-cluster.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2023 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+
+  <bitmap name="WasherControlFeature" type="BITMAP32">
```

Comment:
Name this `Feature` based on us trying to standardize naming. We ended up asking everyone to just use `Feature` (we had some back and forth here)